epics/pv.py

```diff
@@ -253,12 +296,11 @@ def __on_connect(self, pvname=None, chid=None, conn=True):
         self.connected = conn
         return
 
+    @_ensure_context
     def wait_for_connection(self, timeout=None):
         """wait for a connection that started with connect() to finish"""
 
         # make sure we're in the CA context used to create this PV
```

Comment:
Remove stale comment

Reply:
yep, I agree (and sorry for delay.  and thanks for closing all those issues. and thanks in general!)